### PR TITLE
update detection bundle

### PR DESCRIPTION
**Pull request by Can-Zhao**
update config, 
update retrained val metric numbers in readme and meta.json

Fixes # .

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/lung_nodule_ct_detection/configs/metadata.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
     "version": "0.5.4",
```
Comment (yiheng-wang-nv) [inline models/lung_nodule_ct_detection/configs/metadata.json]: here also need to use "0.5.5"

**Review by yiheng-wang-nv — COMMENTED** [review]
Hi @Can-Zhao , I think weights and figures also need to be updated.

**Comment by Can-Zhao** [comment]
> 

Thank you! Yes, I contacted Michael to update figures

**Review by yiheng-wang-nv — APPROVED** [review]

**Pull request merged**